lib/rules/no-obscure-array-access.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Rule = require('./base');
+
+function ERROR_MESSAGE(expression) {
+  return `Obscure expressions are prohibited. ${expression}. Please use Ember's get helper instead. i.e {{get list '0'}} `;
+}
+
+module.exports = class NoObscureArrayAccess extends Rule {
+  visitor() {
+    return {
+      PathExpression(node) {
+        // Regex test for any word character plus digit. i.e.  "test.1"
+        let regex = /\w+.\d/;
+        if (node.original && node.original.match(regex)) {
```

Comment:
could we add fixer for it?

`this.a[1][2][3]` -> `get this.a '1.2.3.'`

Reply:
example: https://astexplorer.net/#/gist/5edec4c14fb7c3554278b8a32b1e3297/ad0139632c9557c891b9d99c56bb51de24caa350

Reply:
Ya, we can definitely make this fixable.